docs/pt_BR/source/conf.py

```diff
@@ -41,49 +48,49 @@
 #
 # source_suffix = ['.rst', '.md']
 source_suffix = '.rst'
-
+source_encoding = 'utf-8-sig'
 # The master toctree document.
 master_doc = 'index'
 
 # General information about the project.
-project = u'correios'
-copyright = u'2017, olist'
-author = u'olist'
+project = 'correios'
+copyright = '2018, olist'
+author = 'olist'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = u'3.1.2'
+version = '3.1.2'
 # The full version, including alpha/beta/rc tags.
-release = u'3.1.2'
+release = '3.1.2'
```

Comment:
it would be better to get the version automatically from setup or configure bumpversion to change version here too if possible